pkg/cmd/roachtest/tests/knex.go

```diff
@@ -111,6 +114,31 @@ func registerKnex(r registry.Registry) {
 		)
 		rawResultsStr := result.Stdout + result.Stderr
 		t.L().Printf("Test Results: %s", rawResultsStr)
+		if err != nil {
+			// We don't have a good way of parsing test results from javascript, so we
+			// do substring matching instead.
+			numFailingRegex := regexp.MustCompile(`There were (\d+) failures`)
+			matches := numFailingRegex.FindStringSubmatch(rawResultsStr)
+			numFailing, convErr := strconv.Atoi(matches[1])
+			if convErr != nil {
+				t.Fatal(convErr)
+			}
+			for _, testName := range []string{
```

Comment:
nit: could we include a note about why these failures are ignored? An inline comment would be perfect.

Reply:
done